### PR TITLE
Localize Swap, Wallet components & Ledger, Dapp connection pages.

### PR DESCRIPTION
**Pull request by 0xDaedalus**
### To Test
- [ ] Connect to a dapp
- [ ] Switch tally ho to default from dapp connection screen
- [ ] Import a ledger - go through the flow 
- [ ] Approve / Swap some tokens
- [ ] Look at the corresponding activities in the Activity list

### CODE REVIEW

File: ui/_locales/en/messages.json [hunk 1]
```diff
@@ -58,6 +66,34 @@
...
+        "stepsExplainer": "Please follow the steps below and click on Try Again!",
+        "step1": "Plug in a single Ledger",
+        "step2": "Enter pin to unlock",
+        "step3": "Open Ethereum App"
+      },
+      "selectDevice": "Select the device",
+      "clickConnect": "Click connect",
+      "previous": "Previous",
+      "next": "Next",
+      "selectLedgeraccounts": "Select ledger accounts",
```
Comment (mhluongo) [inline ui/_locales/en/messages.json:89]: "Ledger" should be capitalized here

File: ui/_locales/en/messages.json [hunk 2]
```diff
@@ -300,8 +336,8 @@
       "spend": "Spend",
       "tokens": "tokens",
       "spendLimit": "Spend limit",
-      "tooltip1": "Spend limit is the amount of funds from a particular asset, that you allow a contract to spend.",
-      "tooltip2": "Infinite tx has the drawback that if the contract is mailicous, it can steal all your funds.",
+      "tooltip1": "Spend limit is the amount of funds from a particular asset that you allow a contract to spend.",
+      "tooltip2": "Infinite tx has the drawback that if the contract is malicous, it can steal all your funds.",
```
Comment (mhluongo) [inline ui/_locales/en/messages.json:340]: s/malicous/malicious/

File: ui/_locales/en/messages.json [hunk 3]
```diff
@@ -324,6 +370,15 @@
       "daoFee": "DAO fee"
     }
   },
+  "switchWallet": {
+    "title": "Want to use another wallet instead?",
+    "tooltip": "You are seeing this because TallyHo! is set as default wallet, you can change this option in the main menu.",
```
Comment (mhluongo) [inline ui/_locales/en/messages.json:375]: "Tally Ho" should have a space and no exclamation point to be consistent across the UI

**Review by mhluongo — COMMENTED** [review]
Quick copy proofing

File: ui/_locales/en/messages.json [hunk 4]
```diff
@@ -310,6 +346,16 @@
...
     "from": "Swap from:",
     "to": "Swap to:",
+    "sellAsset": "You pay",
+    "buyAsset": "You receive",
+    "continueSwap": "Continue Swap",
+    "exchangeRoute": "Exchange route",
+    "rewards": {
+      "header": "Swap rewards for community",
+      "body": "This week, 240,000 DOGGO tokens will be equally shared as Swap Rewards.",
+      "tooltip": "Tally rewards it's users that use swap every week. A council decides weekly prizes and who is eligible.",
```
Comment (jagodarybacka) [inline ui/_locales/en/messages.json:356]:
```suggestion
      "tooltip": "Tally Ho rewards it's users that use swap every week. A council decides weekly prizes and who is eligible.",
```

File: ui/_locales/en/messages.json [hunk 5]
```diff
@@ -324,6 +370,15 @@
       "daoFee": "DAO fee"
     }
   },
+  "switchWallet": {
+    "title": "Want to use another wallet instead?",
+    "tooltip": "You are seeing this because Tally Ho! is set as default wallet, you can change this option in the main menu.",
+    "confirmSwitchWallet": "Yes, switch wallet",
+    "notDefaultWalletMessage": "Tally Ho not Default",
+    "disableWalletExplainer": "We disabled Tally as default wallet for you. You can always enable it back from Menu ☰ at any time.",
```
Comment (jagodarybacka) [inline ui/_locales/en/messages.json:378]:
```suggestion
    "disableWalletExplainer": "We disabled Tally Ho as default wallet for you. You can always enable it back from Menu ☰ at any time.",
```

**Review by greg-nagy — APPROVED** [review]
works well, changes are good 🤩🚀

**Pull request merged**